website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,205 @@
+!!!
+
+#### Size
+
+The Toggle is available in `small`, `medium`, and `large` sizes in both the Ember and Figma components. If used in a paragraph of text, the Ember component will inherit the size from the surrounding text.
+
+![Sizes of the Toggle](/assets/components/rich-tooltip/rich-tooltip-toggle-sizes.png)
+
+#### Icon
+
+An icon can be used in the Toggle in either the `leading` or `trailing` position, or on its own when paired with another element as a label.
+
+![Icon placement of the Toggle](/assets/components/rich-tooltip/rich-tooltip-toggle-icon.png)
+
+#### Underline
+
+Only available as a component in Figma, the ToggleUnderline component can be used **inline** a block of text like a paragraph.
+
+![Underline only toggle](/assets/components/rich-tooltip/rich-tooltip-toggle-underline.png)
+
+### Custom toggle
+
+If absolutely necessary, toggling the Rich Tooltip using a custom element is supported but should be used sparingly and conform to the following usability and accessibility success criteria:
+
+- Must have a minimum target area of 24x24 pixels.
+- Must have a minimum contrast ratio between the background/surface color of 3:1.
+- Should use actionable language or be paired with a label to communicate that additional context is hidden from the user.
+
+More information about this success criteria can be found in the [accessibility](?tab=accessibility) section.
+
+!!! Dont
+
+Don’t use a [Badge](/components/badge) or [BadgeCount](/components/badge-count) as the toggle for the Rich Tooltip, Badges are intentionally non-interactive elements and don't visually communicate that content is hidden behind a `hover`, `click`, or `focus` interaction.
+
+![Example of using the Rich Tooltip with a badge](/assets/components/rich-tooltip/rich-tooltip-trigger-badge-dont.png)
+!!!
+
+!!! Do
+
+Instead, place the Toggle component inline with the Badge to communicate the existence of a Rich Tooltip.
+
+![Example of pairing the default toggle with a Badge](/assets/components/rich-tooltip/rich-tooltip-trigger-badge-do.png)
+!!!
+
+!!! Dont
+
+Don’t toggle a Rich Tooltip from a form element like a [Text Input](/components/form/text-input), [Select](/components/form/select), [Textarea](/components/form/textarea), or form primitive like a [Label](/components/form/primitives#formlabel) or [HelperText](/components/form/primitives#helpertext).
+
+![Example of toggling a Rich Tooltip from a form element](/assets/components/rich-tooltip/rich-tooltip-trigger-form-input-dont.png)
+
+The following example using HelperText will not be read out as interactive to users with screen readers, only the text will be read. More details on this recommendation can be found in the [Helper Text documentation](/components/form/primitives?tab=code#formhelpertext-1).
+
+![Example within Helper Text](/assets/components/rich-tooltip/rich-tooltip-trigger-helper-text-dont.png)
+!!!
+
+!!! Do
+
+Instead, include additional details in a [Reveal](/components/reveal).
```

Comment:
@jorytindall so we're not suggesting anymore to use an icon next to the form label?

Reply:
Wouldn't this have the same problems with accessibility as including it in the helper text? I had initially included that example, but within the helper text only the text will be read to a screen reader, which in the case of an icon-only variant would that be the aria label?

I did a quick test of this in the showcase, and I'm not really sure that what we outlined for interactive elements [here](https://helios.hashicorp.design/components/form/primitives?tab=code#formhelpertext-1) is true. I tested with voiceover and for the icon paired with the label it read the "description" as the "generic content" within the bubble, and the same for the helper text and noted that the button was expanded. However, in testing the example linked above, only the text of the link is read out. Are links treated differently in this case than buttons? Can you confirm this as well? If this is the case, then I think we could include both of these examples, unless I'm missing something.

cc. @MelSumner

Reply:
I'm pretty sure the content of the popover won't be read out _naturally_, the user with a screen reader has a few more ways of exploring content using arrow keys when the SR is turned on.